charts/sn-platform/values.yaml

```diff
@@ -158,36 +158,36 @@ images:
   #   pullPolicy: IfNotPresent
   autorecovery:
     repository: streamnative/sn-platform
-    tag: "2.8.1.30"
+    tag: "2.9.2.5"
     pullPolicy: IfNotPresent
   broker:
     repository: streamnative/sn-platform
-    tag: "2.8.1.30"
+    tag: "2.9.2.5"
     pullPolicy: IfNotPresent
   proxy:
     repository: streamnative/sn-platform
-    tag: "2.8.1.30"
+    tag: "2.9.2.5"
     pullPolicy: IfNotPresent
   pulsar_detector:
     repository: streamnative/sn-platform
-    tag: "2.8.1.30"
+    tag: "2.9.2.5"
     pullPolicy: IfNotPresent
   functions:
     repository: streamnative/sn-platform
-    tag: "2.8.1.30"
+    tag: "2.9.2.5"
     pullPolicy: IfNotPresent
   function_worker:
     repository: streamnative/sn-platform
-    tag: "2.8.1.30"
+    tag: "2.9.2.5"
     pullPolicy: IfNotPresent
   # NOTE: allow overriding the toolset image
   toolset:
     repository: streamnative/sn-platform
-    tag: "2.8.1.30"
+    tag: "2.9.2.5"
```

Comment:
Can you update the image version to 2.9.2.9, which includes a fix for oidc token